web/server/vue-cli/config/webpack.dev.js

```diff
@@ -45,21 +44,55 @@ module.exports = merge(common, {
         }
       ]
     },
-    proxy: [{
-      context: [
-        ...CC_SERVICE_ENDPOINTS.map(endpoint => `**/${endpoint}`),
-        "/docs/**"
-      ],
-      target: CC_THRIFT_API_HOST + ':' + CC_THRIFT_API_PORT,
-      changeOrigin: true,
-      secure: false
-    }]
+    proxy: {
```

Comment:
It would be better from maintenance perspective that we dont duplicate these endpoint names. Please use the above shorter form if possible. 
target: CC_THRIFT_API_HOST + ':' + CC_THRIFT_API_PORT,